boards/avr-rss2/doc.txt

```diff
@@ -0,0 +1,106 @@
+#Getting Started with RIOT using the avr-rss2 board
+#Document version
+V1.3 2019-11-07
+
+# Overview
+This guide's aim is to help you start using RIOT for RSS2 boards but
+platform can support different AtMega-RF boards based on AtMega256RFR2
+
+The guide assumes that you have basic understanding of how to use the command
+line and perform basic admin tasks on UNIX family OSs. You should also have
+understanding about the RIOS-OS design as well have C programming skills.
+Directory `$(RIOTBASE)/boards/avr-rss2
+
+#Board Features
+* Chip Antenna. Supercardiod.
+* Robust radio performance. Up to 300 meter line-of-sight.
+* Unique EUI64 address via 24MAC602 chip with 2k EEPROM
+* Rev 2.4 Onboard BME280 for Temp/RH/Pressure
+* Light Sensor.
+* 32kHz RTC clock xtal
+* HW comparator chip and input. 
+* Progammable power FET, (relay) for power on/off of sensors.
```

Comment:
Codespell should report this king of typo: progammable => programmable. There's another one on the same word below.